tests/integration/ha_tests/test_async_replication.py

```diff
@@ -315,24 +305,22 @@ async def test_promote_standby(
             ),
         )
 
+    logger.info("rerelate test app")
+    await ops_test.model.applications[DATABASE_APP_NAME].remove_relation(
+        "database", f"{APPLICATION_NAME}:{FIRST_DATABASE_RELATION_NAME}"
+    )
+    await ops_test.model.wait_for_idle(
+        apps=[DATABASE_APP_NAME, APPLICATION_NAME], status="active", raise_on_blocked=True
+    )
+    await ops_test.model.relate(
+        DATABASE_APP_NAME, f"{APPLICATION_NAME}:{FIRST_DATABASE_RELATION_NAME}"
+    )
+    await ops_test.model.wait_for_idle(
+        apps=[DATABASE_APP_NAME, APPLICATION_NAME], status="active", raise_on_blocked=True
+    )
+
```

Comment:
DB is owned by the user of the test app in the other model, so the test app in this model loses access to the public schema. Rerelating to change ownership. Not sure why this isn't failing on VM.